src/components/pages/ComparisonPage/ComparisonRow.tsx

```diff
@@ -13,22 +17,25 @@ function ComparisonRow({ children }: Props) {
   const emptyCellsText = (
     <Translate nb="Du kan søke for å legge til flere kolonner" />
   );
-  return (
-    <div className={`${styles.row_style}`}>
-      {children}
 
-      <div className={`${styles.empty_cells_placeholder}`}>
-        <div>{emptyCellsText}</div>
-      </div>
-      <div className={`${styles.empty_cells_placeholder}`}>
-        <div>{emptyCellsText}</div>
-      </div>
-      <div className={`${styles.empty_cells_placeholder}`}>
-        <div>{emptyCellsText}</div>
-      </div>
-      <div className={`${styles.empty_cells_placeholder}`}>
+  const emptyCellsLength =
+    window.innerWidth < 768
```

Comment:
Du må bruke `num_compare_sizing` for å sørge for at dette blir oppdatert når brukeren endrer størrelsen på vinduet